torch/masked/maskedtensor/core.py

```diff
@@ -290,12 +293,12 @@ def _preprocess_data(self, data, mask):
                 data = _sparse_csr_where(mask, data, torch.tensor(0))
 
         # Have to pick awkward names to not conflict with existing fields such as data
-        self._masked_data = data
-        self._masked_mask = mask
+        self._masked_data = data.clone()
+        self._masked_mask = mask.clone()
 
     def _validate_members(self):
-        data = self._masked_data
-        mask = self._masked_mask
+        data = self.get_data()
```

Comment:
`.get_data()` is now differentiable, so I don't think you want to use it here?

Reply:
fixed!